ledger-core/application/cmd/insolar/create_default_configs.go

```diff
@@ -12,8 +12,8 @@ import (
 	"gopkg.in/yaml.v2"
 
 	"github.com/insolar/assured-ledger/ledger-core/application/bootstrap"
-	pulsewatcher "github.com/insolar/assured-ledger/ledger-core/cmd/pulsewatcher/config"
 	"github.com/insolar/assured-ledger/ledger-core/configuration"
+	configuration2 "github.com/insolar/assured-ledger/ledger-core/testutils/pulsewatcher/configuration"
```

Comment:
`configuration2`?

Reply:
I'm not sure, should I put puslewatcher configuration to common configuration, therefore it has it's own configuration package